### PR TITLE
[dev-v2.7] wins-upgrader annotation fix

### PR DESCRIPTION
**Pull request by doflamingo721**
The rancher-wins-upgrader chart version `0.0.100` is visible on Rancher v2.7 because of the missing rancher-version constraints. This PR adds the rancher-version annotation to hide the chart from 2.7.
For more context refer: https://github.com/rancher/rancher/issues/38965#issuecomment-1268131263

Linked issue:
https://github.com/rancher/rancher/issues/38965

### CODE REVIEW

**Comment by eliyamlevy** [comment]
I think we need this shown in the patch files as well no @aiyengar2? Did you run make patch when making these changes?

File: charts/rancher-wins-upgrader/0.0.100/Chart.yaml [hunk 1]
```diff
@@ -3,6 +3,7 @@ annotations:
   catalog.cattle.io/namespace: cattle-wins-system
   catalog.cattle.io/os: windows
   catalog.cattle.io/release-name: rancher-wins-upgrader
+  catalog.cattle.io/rancher-version: '>= 2.5.0-0 <= 2.5.99-0'
```
Comment (PennyScissors) [inline charts/rancher-wins-upgrader/0.0.100/Chart.yaml:5]: Can you change this to `'>= 2.5.0-0 < 2.6.0-0'`

**Comment by doflamingo721** [comment]
@eliyamlevy This chart does not exist in the packages directory because it was no longer needed in 2.7 since rke1 windows is no longer supported and rke2 does not need this chart. Please refer to this [comment](https://github.com/rancher/charts/pull/2130#issuecomment-1265905310).

File: charts/rancher-wins-upgrader/0.0.100/Chart.yaml (hunk 1 above)
Comment (doflamingo721) [reply charts/rancher-wins-upgrader/0.0.100/Chart.yaml]: Updated the annotation. Can you please review again?

**Review by pennyscissors — APPROVED** [review]

**Pull request merged**